admin/migrations/0004-add-prayer-fuel-meta.php

```diff
@@ -13,7 +13,7 @@ public function up() {
         $selected_campaign_id = get_option( "pray4ramadan_selected_campaign" );
         $selected_campaign = DT_Campaign_Settings::get_campaign( $selected_campaign_id );
 
-        if ( !$selected_campaign["start_date"]["formatted"] ) {
+        if ( !isset( $selected_campaign["start_date"]["formatted"] ) ) {
```

Comment:
Thanks Jon.  I had been trying to get my debugger to work to check what this looked like in the code...

I thought that it was always there, but sometimes was set to null.... But `isset` does that, so that is good :)

Reply:
If there is no campaign selected, then `$selected_campaign` will be empty